src/routes/index.js

```diff
@@ -1 +1,59 @@
 // ADD CODE HERE
+var express = require('express');
+const bodyParser = require('body-parser')
+const { getAllTodos, getCompletedTodos,completeOneTodo,addOneTodo } = require('./routes/index.js')
+const app = express()
+app.set('view engine', 'pug')
+app.use(express.static('public'))
+app.use(bodyParser.json())
+app.use(bodyParser.urlencoded({ extended: true}))
+
+
+app.post('/complete', (request, response) => {
+  const id = request.body.id
+  completeOneTodo(id)
+  .then((todo) => {
+    response.redirect('/')
+  })
+})
+
+
+app.post('/add', (request, response) => {
+  const { name, description } = request.body
+  addOneTodo(name, description)
+  .then((todo) => {
```

Comment:
dont need to put `todo` as an argument in this callback  since you are just redirecting.
lines 24-25 can be:
```
.then( () => {
  response.redirect('/')
}
```